packages/sanity/src/core/bundles/components/panes/BundleActions.tsx

```diff
@@ -1,52 +1,61 @@
 import {AddIcon, CheckmarkIcon} from '@sanity/icons'
 import {useToast} from '@sanity/ui'
-import {type ReactNode, useCallback, useEffect, useState} from 'react'
+import {type ReactNode, useCallback, useState} from 'react'
 import {filter, firstValueFrom} from 'rxjs'
 import {useDocumentOperation, useDocumentStore, useTranslation} from 'sanity'
 
 import {Button} from '../../../../ui-components'
 import {type BundleDocument} from '../../../store/bundles/types'
-import {versionDocumentExists} from '../../util/util'
+import {getBundleSlug} from '../../util/util'
 
 interface BundleActionsProps {
   currentGlobalBundle: BundleDocument
+  /**
+   * Base document id, the published one
+   */
   documentId: string
```

Comment:
for the sake of being super clear, would it make sense to name this `publishedId`?